src/t_string.c

```diff
@@ -73,7 +73,9 @@ void setGenericCommand(client *c, int flags, robj *key, robj *val, robj *expire,
         if (getGenericCommand(c) == C_ERR) return;
     }
 
-    found = (lookupKeyWrite(c->db,key) != NULL);
+    dictEntry *de = dbFind(c->db, key->ptr);
+    robj *o = lookupKeyWriteWithDictEntry(c->db,key,de);
+    found = (o != NULL);
```

Comment:
```suggestion
    found = (lookupKeyWriteWithDictEntry(c->db,key,de) != NULL);
```

i don't see `o` being used again.